tests/compiler/nonNullAssertion.untouched.wat

```diff
@@ -54,14 +56,18 @@
   i32.shr_u
   i32.lt_u
   if (result i32)
```

Comment:
Also increase count of locals and redundancy read/write for `Array#__get`

Reply:
Increased count of locals is expected. There was some aliasing going on previously, where if an argument to an inlined function was a `local.get`, instead of creating a new temp. local, it would alias it. That was wrong, though, because it would modify the original local when set, so there is one local per argument now.